WcaOnRails/spec/models/competition_spec.rb

```diff
@@ -23,6 +23,23 @@
     end
   end
 
+  context "delegates" do
+    it "delegates for future comps must be delegates" do
```

Comment:
Yeah... how about distinguishing between "delegates" and "the users marked as delegates for this competition"? I'm thinking something like this:

> "users marked as competition delegates must be delegates"

Let me know if this discussion has reached your threshold for pedanticness. At this point, I'm okay with "delegates for future comps must be valid" (as per your original suggestion), but I'm curious if we can do better.